ballerina-command/src/main/resources/cli-help/ballerina-pull.help

```diff
@@ -1,14 +1,16 @@
 NAME
-       ballerina-dist-pull - Pull and use a given distribution version
+       ballerina-dist-pull - Fetch a given distribution and set it as the
+                             active version
 
 SYNOPSIS
        ballerina dist pull <distribution-version>
 
 
 DESCRIPTION
-       Dist-pull pulls and uses a given Ballerina version from the remote repository.
+       Dist-pull fetches a given Ballerina version from the remote repository
+       and sets it as the active version.
 
 
 EXAMPLES
-       Pull and use Ballerina 1.0.0.
+       Fetch and use Ballerina 1.0.0.
```

Comment:
@MaryamZi We have another command called "fetch" in the spec (although we removed it). Can't we keep pull or may be any other term like download?

Reply:
Since pull is pretty much fetch + use, and we are mentioning use too, I guess this is OK?